### PR TITLE
transfer little batch

### PR DESCRIPTION
**Pull request by JackTan25**
## What type of PR is this?

- [ ] API-change
- [x] BUG
- [x] Improvement
- [ ] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

## Which issue(s) this PR fixes:

issue #8294 #8298

## What this PR does / why we need it:

### CODE REVIEW

File: pkg/vm/engine/disttae/txn.go [hunk 1]
```diff
@@ -140,9 +142,103 @@ func (txn *Transaction) WriteBatch(
...
+			container, tbl, err := txn.getContainer(key)
+			if err != nil {
+				return err
+			}
+			container.InitBuffers(mp[key][0], 0)
+			for i := 0; i < len(mp[key]); i++ {
+				container.Put(mp[key][i], 0)
+			}
+			old_ctx := txn.proc.Ctx
+			txn.proc.Ctx = context.Background()
```
Comment (nnsgmsone) [inline pkg/vm/engine/disttae/txn.go:181]: Why do you need this?

Comment (JackTan25) [reply pkg/vm/engine/disttae/txn.go:181]: fix context cancel bug

Comment (nnsgmsone) [reply pkg/vm/engine/disttae/txn.go:181]: This must be a bug, the upper layer passed context should not be in this case timeout, far from the timeout time, and did not exit.

Comment (JackTan25) [reply pkg/vm/engine/disttae/txn.go:181]: #8328，this will be fixed by people, shoule be pengzheng

**Review by aressu1985 — APPROVED** [review]

**Review by w-zr — APPROVED** [review]

**Review by zhangxu19830126 — APPROVED** [review]

**Review by nnsgmsone — APPROVED** [review]

**Review by m-schen — APPROVED** [review]

**Comment by JackTan25** [comment]
@matrix-meow rerun failed

**Pull request merged**